bundles/org.openhab.binding.freemobilesms/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This looks like an old classpath file. Look at other bindings and see if you copy a file from another binding.